internal/cli/opsmanager/backup/restores_list.go

```diff
@@ -40,27 +45,30 @@ func (opts *RestoresListOpts) initStore() error {
 
 func (opts *RestoresListOpts) Run() error {
 	listOpts := opts.NewListOptions()
-	r, err := opts.store.ContinuousRestoreJobs(opts.ConfigProjectID(), opts.clusterName, listOpts)
+	r, err := opts.store.ContinuousRestoreJobs(opts.ConfigProjectID(), opts.clusterID, listOpts)
 	if err != nil {
 		return err
 	}
 
-	return output.Print(config.Default(), "", r)
+	return output.Print(config.Default(), restoresTemplate, r)
 }
 
-// mongocli atlas backup(s) restore(s) job(s) list <clusterName|clusterID> [--page N] [--limit N]
+// mongocli atlas backup(s) restore(s) job(s) list <clusterId> [--page N] [--limit N]
 func RestoresListBuilder() *cobra.Command {
 	opts := new(RestoresListOpts)
 	cmd := &cobra.Command{
-		Use:     "list <clusterName|clusterID>",
+		Use:     "list <clusterId>",
```

Comment:
for atlas you can use a `clusterName` but we kind of stopped supporting atlas given that this kind of back up is deprecated